### DISCUSSION TITLE
SVG sequence

### DISCUSSION THREAD

**Discussion by Faktor-de-Voure** [mbasaglia/glaxnimate#2] [General]
Is there a good solution to export a SVG Sequence or convert exported SMIL-SVG to sequence of SVG files?

Regards!

**Comment by mbasaglia** [comment]
Hmm, at the moment you can export individual frames as SVG, so it can be done but it's a bit tedious.

It shouldn't be too hard to add some kind of functionality that exports image sequences, perhaps it could work as a dialog that selects the export format and frequency of exported frames.

**Comment by Faktor-de-Voure** [comment]
> Hmm, at the moment you can export individual frames as SVG, so it can be done but it's a bit tedious.

But it's still better than bodymovin's svg export in Ae.
> perhaps it could work as a dialog that selects the export format and frequency of exported frames.

Oh, yes. Snippets. Don't recognize it at first time. I found render_image funtion, but it return PIL, a bitmap. Not a vector. Which one for vectors is?

**Reply by mbasaglia** [reply]
You can use `glaxnimate.io.SvgFormat.render_frame()`

**Reply by Faktor-de-Voure** [reply]
Oh, thanks!

And I have one more question, but _quite_ out of topic.
Is that possible to fit animation along to path, or it can be achieve by edit exported svg?

**Reply by mbasaglia** [reply]
I'm not sure I understand the question

**Reply by Faktor-de-Voure** [reply]
Smth like this.
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/animateMotion

**Reply by mbasaglia** [reply]
Ah that, it isn't implemented yet, but it will at some point

**Reply by Faktor-de-Voure** [reply]
Pity. But good luck with that.
And center pivot point will be soon, or not? It seems to be useful. "Free transform" currently is... quite painful without that.

**Reply by mbasaglia** [reply]
There's already an anchor point property for transforms

**Reply by Faktor-de-Voure** [reply]
I mean smth like this.
https://imgur.com/a/cGDmwY5

**Reply by mbasaglia** [reply]
Ah ok, it should be easy enough
